docs/modules/secret-operator/pages/secretclass.adoc

```diff
@@ -38,6 +38,26 @@ CAUTION: Attributes of the certificate (such as the expiration date, fingerprint
 
 xref:scope.adoc[Scopes] are used to populate the claims (such as `subjectAlternateName`) of the provisioned certificates.
 
+[#backend-tls-key-length]
+=== `TLS key length`
```

Comment:
note: TLS doesn't have a "key length", it is just a protocol. The underlying key pair used to sign the certificate has a key length.

Reply:
tried to adapt here https://github.com/stackabletech/secret-operator/pull/506/commits/5506a75aca6c5e4dd16857ac2b3abd321f041089

Reply:
If we would call it "TLS certificate key pair generation", I would be happy for now.

Reply:
done https://github.com/stackabletech/secret-operator/pull/506/commits/85ab8c46d2fea28acabc537f8454cba8ab9a5143